device/jsonld-contexts/device.jsonld

```diff
@@ -7,13 +7,15 @@
         "batteryVoltage": "https://vocab.egm.io/batteryVoltage",
         "category": "https://schema.org/category",
         "chargerStatus": "https://vocab.egm.io/chargerStatus",
+        "co2": "https://vocab.egm.io/co2",

```

Comment:
these are "business" attributes, they should not appear in a device model

Reply:
It's OK, I don't really need them. It's just ugly in the Device management module in twin picks because it is usecase agnostic and so they appear expanded. 

I agree it's not good solution either to put these attrs here. Closing.